<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_properties">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="_nameEdit">
         <property name="placeholderText">
          <string>Projet name (required)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="parentLabel">
         <property name="text">
          <string>Parent project:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ProjectTreeComboBox" name="_parentBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="hourlyRateLabel">
         <property name="text">
          <string>Hourly rate:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="_hourlyRateEdit">
         <property name="placeholderText">
          <string>e.g. 42.50 USD</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="noteRequiredLabel">
         <property name="text">
          <string>Notes on entries:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="TriStateFlagComboBox" name="_noteRequired" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTreeComboBox</class>
   <extends>QComboBox</extends>
   <header>commonWidgets/ProjectTreeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>TriStateFlagComboBox</class>
   <extends>QWidget</extends>
   <header>commonWidgets/TriStateFlagComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
